frontend/package.json

```diff
@@ -50,6 +50,10 @@
     "webpack": "^4.42.1",
     "webpack-cli": "^3.3.11"
   },
+  "peerDependencies": {
```

Comment:
So I haven't taken the time to take a deep look at this yet, but I just tried to remove one of these peer dependencies block and get these warnings when running Yarn as a result:

```
➤ YN0000: ┌ Resolution step
➤ YN0002: │ tracim_app_admin_workspace_user@workspace:frontend_app_admin_workspace_user doesn't provide cheerio@0.19.x || 0.20.x || 0.22.x || ^1.0.0-0 requested by chai-enzyme@npm:1.0.0-beta.1
➤ YN0002: │ tracim_app_admin_workspace_user@workspace:frontend_app_admin_workspace_user doesn't provide cheerio@^1.0.0 requested by tracim_frontend_lib@workspace:frontend_lib
```

I wanted to check that this was still necessary in Yarn 2 since I made those changes with Yarn 1 (because of mistake).


Putting them in `dependencies` or `devDependencies` yields:

```
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: No candidate found for cheerio@npm:^1.0.0
    at C (/home/raph/dl/tracim/.yarn/releases/yarn-berry.js:42:105034)
    at process._tickCallback (internal/process/next_tick.js:68:7)
➤ YN0000: └ Completed in 0.35s
➤ YN0000: Failed with errors in 0.35s
```

We probably shouldn't put them in `dependencies` or `devDependencies` anyway even if we could, to make it clear that our code does not directly depend on them.

We should both take a look at how peer dependencies work. I'm not sure if we should block this pull request in the meantime tough. What do you think?

Reply:
So I haven't taken the time to take a deep look at this yet, but I just tried to remove one of these peer dependencies block and get these warnings when running Yarn as a result:

```
➤ YN0000: ┌ Resolution step
➤ YN0002: │ tracim_app_admin_workspace_user@workspace:frontend_app_admin_workspace_user doesn't provide cheerio@0.19.x || 0.20.x || 0.22.x || ^1.0.0-0 requested by chai-enzyme@npm:1.0.0-beta.1
➤ YN0002: │ tracim_app_admin_workspace_user@workspace:frontend_app_admin_workspace_user doesn't provide cheerio@^1.0.0 requested by tracim_frontend_lib@workspace:frontend_lib
```

I wanted to check that this was still necessary in Yarn 2 since I made those changes with Yarn 1 (because of mistake).


Putting them in `dependencies` or `devDependencies` yields:

```
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: No candidate found for cheerio@npm:^1.0.0
    at C (/home/raph/dl/tracim/.yarn/releases/yarn-berry.js:42:105034)
    at process._tickCallback (internal/process/next_tick.js:68:7)
➤ YN0000: └ Completed in 0.35s
➤ YN0000: Failed with errors in 0.35s
```

We probably shouldn't put them in `dependencies` or `devDependencies` anyway even if we could, to make it clear that our code does not directly depend on them.

We should both take a look at how peer dependencies work. I'm not sure if we should block this pull request in the meantime tough. What do you think?